src/filters/AIHelp.ts

```diff
@@ -0,0 +1,58 @@
+import { Filter } from "./filter";
+import { Message, MessageEmbed } from "discord.js";
+import {Wit, log} from 'node-wit';
+import * as config from "../config.json";
+export class AIHelp extends Filter {
+    constructor() {
+        super(9);
+    }
+    async run(message: Message): Promise<boolean> {
+        if (message.author.bot) {
```

Comment:
actually you can remove this if and instead of else `if (!message.author.bot) `. After this you will have only one return true